secure-jwt/apps/expenses/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+mp.jwt.verify.issuer = https://example.com/redhattraining
+smallrye.jwt.sign.key.location = ${HOME}/DO378/secure-jwt/privateKey.pem
```

Comment:
is there a way to refer to the certs without requiring the app to be in the `~/DO378/secure-jwt` dir?

Reply:
Not sure about this. Another option I considered is to store the key files in `~/.certs` or similar, but I decided to keep them in the app directory to keep everything under `~/DO378`.

Reply:
I'll add a note in the README file, just in case someone wants to play with the app.